docs/mixpanel.md

```diff
@@ -0,0 +1,15 @@
+### What is Mixpanel?
+It is an analytic tool that gets you insighst on how users interact with your digital product. These insights will help enable you to make decisions on how your users can get the best out of your digital productand and eventually retain them.
+
+### How to set it up?
+1. Create an account ( sign up with SSO or email and password)
+2. Go to settings
+    - Get your access tokens ( Project token and if needed API token)
+3. Create a project ( React project with create-react-app for example )
```

Comment:
I think we can leave this out because we are going to assume they are implementing it with THIS project, so you can remove this step.